pipeline/ingest/loaders/vincent.py

```diff
@@ -212,5 +218,76 @@ def load_tracking(self, session_dir, subject_name, session_datetime):
                  'tracking_files': [tracking_fp.relative_to(self.root_data_dir)],
                  'WhiskerTracking': [{'whisker_idx': wid, **wdata} for wid, wdata in whiskers.items()]}]
 
-    def load_ephys(self):
-        pass
+    def load_ephys(self, session_dir, subject_name, session_basename):
+        analysis_dir = session_dir / 'Analysis' / f'{subject_name}_{session_basename}'
+        if not analysis_dir.exists():
+            raise FileNotFoundError(f'{analysis_dir} not found!')
+
+        # Expect 3 files per probe: .spikes.mat; recInfo.mat; .prb
+        # As an example, only expect one probe per session
+        jrclust_fp = list(analysis_dir.glob('*.spikes.mat'))
+        recinfo_fp = list(analysis_dir.glob('*recInfo.mat'))
+        prb_adaptor_fp = list(analysis_dir.glob('*.prb'))
+
+        if len(jrclust_fp) != 1:
+            raise FileNotFoundError(f'Unable to find one JRCLUST file - Found: {jrclust_fp}')
+        if len(recinfo_fp) != 1:
+            raise FileNotFoundError(f'Unable to find one Recording Info file - Found: {recinfo_fp}')
+        if len(prb_adaptor_fp) != 1:
+            raise FileNotFoundError(f'Unable to find one Probe Adapter file - Found: {prb_adaptor_fp}')
+
+        # read recInfo file
+        recInfo = {}
+        with h5py.File(str(recinfo_fp[0]), mode='r') as f:  # ephys file
+            info = f['recInfo']
+            recInfo['fs'] = info['samplingRate'][0][0]
+            recInfo['channel_num'] = info['numRecChan'][0][0]
+
+            rec_date = str().join(chr(c) for c in info['date'])
+            recInfo['recording_time'] = datetime.strptime(rec_date, '%d_%b_%Y_%H_%M_%S')
+            recInfo['recording_system'] = str().join(chr(c) for c in info['sys'])
+
+        # read adapter file
+        adapter = {}
+        with open(prb_adaptor_fp[0], mode='r') as f:
+            for line in f.readlines():
+                if line.startswith('%') or line.startswith('\n'):
+                    continue
+                split_vals = line.split('=')
+                k = split_vals[0]
+                v = split_vals[1]
+                try:
+                    v_str = re.match('\[.*\]', v.strip()).group()
+                    adapter[k.strip()] = ast.literal_eval(v_str.replace(' ', ','))
+                except:
+                    adapter[k.strip()] = v.strip()
+
+        channel_map = np.array(adapter['channels'])
+
+        # read JRCLUST results
+        jrclust = JRCLUST(jrclust_fp[0])
+
+        # probe type
+        probe_type, _, adapter_type = prb_adaptor_fp[0].stem.split('_')
```

Comment:
```suggestion
        probe_id = sessinfo['ephys']['probe']
        adapter_type = sessinfo['ephys']['adapter']
```
 pass along probe_id to ephys_ingest? Need info from lab.Probe to interpret further, or adjust exported info